scripts/updateReleaseNotes/init.ts

```diff
@@ -41,12 +42,10 @@ if (argv.token === 'your token here') {
   process.exit(1);
 }
 
-const repoDetails = {
+export const repoDetails = {
   owner: argv.owner,
   repo: argv.repo,
 };
 
 // Authenticate with github and set up logging if debug arg is provided
-const github = new GitHubApi({ ...(argv.debug ? { log: console } : {}), auth: 'token ' + argv.token });
```

Comment:
pass `token` instead of `auth` similar, keep it the same as the other place

Reply:
Not sure what you mean?

Reply:
This script has always taken an auth token as a command line argument. But I couldn't use that pattern in the beachball config.

Reply:
oh i meant if we can change here to 
```
const github = new GitHubApi({ ...(argv.debug ? { log: console } : {}), token:  argv.token })
```

pass `token` instead of `auth` (assuming they are the same?)

Reply:
Hmm it looks like `token` isn't actually valid. I'll change the new one to also use `auth`.